src/components/doc-explorer/TypeDetails.tsx

```diff
@@ -61,22 +62,3 @@ export default class TypeDetails extends React.Component<TypeDetailsProps> {
     );
   }
 }
-
```

Comment:
It's ok to use internal components right?
Why you moved it into a separate file?
Anyway, such changes should be done in separate PR.

Reply:
Done. Moved into separate pull request #159.

Reply:
I've spent some time to find the `EnumValue` declaration. Other classes are declared in the separate files with the same name as a class. That's why I've refactored it into separate file.